### ISSUE TITLE
CT: Unrecognized record-and-playback mode

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/neuron#148]
neuron has been failing CT since 4/19/22 @ 7:37pm.

```
neuron : fuzz : unbuilt
https://bayes.colorado.edu/continuous-testing/ct-snapshots/1650478991252/neuron/neuron_en.html?continuousTest=%7B%22test%22%3A%5B%22neuron%22%2C%22fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1650478991252%22%2C%22timestamp%22%3A1650483517622%7D&brand=phet&ea&fuzz&memoryLimit=1000
Query: brand=phet&ea&fuzz&memoryLimit=1000
Uncaught Error: Assertion failed: Neuron Model updateSimAndPlaybackParticleVisibility Error: Unrecognized record-and-playback mode.
Error: Assertion failed: Neuron Model updateSimAndPlaybackParticleVisibility Error: Unrecognized record-and-playback mode.
at window.assertions.assertFunction (https://bayes.colorado.edu/continuous-testing/ct-snapshots/1650478991252/assert/js/assert.js:25:13)
at assert (NeuronModel.js:852:16)
at updateSimAndPlaybackParticleVisibility (NeuronModel.js:442:9)
at listener (Property.ts:386:4)
at link (NeuronModel.js:260:22)
at (NeuronScreen.js:21:41)
at createModel (Screen.ts:276:23)
at initializeModel (Sim.ts:833:15)
at (Sim.ts:844:10)
id: Bayes Chrome
Snapshot from 4/20/2022, 12:23:11 PM
```

Running locally, I see the same problem. The sim fails at startup with this stack trace:

```
assert.js:25 Uncaught Error: Assertion failed: Neuron Model updateSimAndPlaybackParticleVisibility Error: Unrecognized record-and-playback mode.
    at window.assertions.assertFunction (assert.js:25:13)
    at NeuronModel.updateSimAndPlaybackParticleVisibility (NeuronModel.js? [sm]:852:17)
    at NeuronModel.updateRecordPlayBack (NeuronModel.js? [sm]:442:10)
    at Property.link (Property.ts:386:5)
    at new NeuronModel (NeuronModel.js? [sm]:260:23)
    at NeuronScreen.backgroundColorProperty [as createModel] (NeuronScreen.js? [sm]:21:42)
    at NeuronScreen.initializeModel (Screen.ts:276:24)
    at Array.<anonymous> (Sim.ts:833:16)
    at Sim.ts:844:11
```

`NeuronScreen.backgroundColorProperty [as createModel]` looks very odd.

**Comment by pixelzoom** [comment]
@jbphet When this issue is resolved, you'll need to verify changes to TimeSpeed for https://github.com/phetsims/scenery-phet/issues/736.  I was unable to verify those changes because this sim would not start.  I would have waited to push, but TimeSpeed was a large amount of work, and I didn't get any reply from you on Slack.

**Comment by jbphet** [comment]
I've fixed the issue.  There was a bad assertion statement in the code that has been there pretty much forever, and it was corrected but apparently not verified in https://github.com/phetsims/neuron/commit/3f234b4bf7b4612aa0bf1f077bd36ad4fcd16f5d.  Once I fixed that up, there was a problem with the declaration of `NeuronClockModelAdapter.timeSpeedProperty`, so I corrected that as well.  I tested the basic functionality of the sim and did a few minutes of fuzz testing as well, and everything passed.  I'll close this, but will also check CT over the next few days to make sure this sim is not failing.
